modules/apps/object/object-admin-rest-impl/src/main/java/com/liferay/object/admin/rest/internal/dto/v1_0/util/ObjectDefinitionSettingUtil.java

```diff
@@ -35,41 +38,89 @@ public class ObjectDefinitionSettingUtil {
 						objectDefinitionSettingLocalService.
 							createObjectDefinitionSetting(0L);
 
-				if (!StringUtil.equals(
+				if (StringUtil.equals(
+						ObjectDefinitionSettingConstants.
+							NAME_ACCEPTED_GROUP_IDS,
+						objectDefinitionSetting.getName()) ||
+					StringUtil.equals(
+						ObjectDefinitionSettingConstants.
+							NAME_ROOT_OBJECT_DEFINITION_IDS,
+						objectDefinitionSetting.getName())) {
+
+					return _setObjectDefinitionsSettingsByExternalReferenceCode(
+						serviceBuilderObjectDefinitionSetting,
+						objectDefinitionSetting);
+				}
```

Comment:
@victorKammerer there will be other settings different of `NAME_ACCEPTED_GROUP_IDS` and `NAME_ROOT_OBJECT_DEFINITION_IDS`, for this situation let's worry about the future of this part of the code, so basically invert your conditional to:

````java
if (!StringUtil.equals(
		ObjectDefinitionSettingConstants.NAME_ACCEPTED_GROUP_IDS,
		objectDefinitionSetting.getName()) &&
	!StringUtil.equals(
		!ObjectDefinitionSettingConstants.NAME_ROOT_OBJECT_DEFINITION_IDS,
		objectDefinitionSetting.getName())) {
````